// See README.txt for information and build instructions.
syntax = "proto2";

package msg;

message ReadyRequest {
   required string token = 1;
}

message ReadyResponse {
   required bytes result = 1;
}
//find other user, return channel  
message EnterBattleResponse {
   required string channel = 1;
}

